.github/workflows/vroom.yml

```diff
@@ -0,0 +1,26 @@
+name: Vroom CI
+on:
+  push:
```

Comment:
Same on tag build than in the other workflow.

Reply:
Same as above.